### DISCUSSION TITLE
importerror with pytorch_lightning.utilities.cli import CALLBACK_REGISTRY

### DISCUSSION THREAD

**Discussion by bparaj** [Lightning-AI/pytorch-lightning#16428] [General]
I was moving from pytorch-lightning `1.6.5` to `1.8.4`. I encountered the following:

```python
    from pytorch_lightning.utilities.cli import CALLBACK_REGISTRY
E   ImportError: cannot import name 'CALLBACK_REGISTRY' from 'pytorch_lightning._graveyard.cli' (/usr/src/ai/pkgs/pytorch_lightning/_graveyard/cli.py)
```
I couldn't find documentation related to how to handle this change.

**Comment by mauvilsa** [comment]
The cli registries were deprecated in #13221 and from `1.7.0` you would see [deprecation warnings](https://github.com/Lightning-AI/lightning/pull/13221/files#diff-9b9744f64d4b698e42b95f5a62f92e859428e85d2b78d79dec4bb330e2ca60b7). But you might be making a mistake and not really using `1.8.4`, since `CALLBACK_REGISTRY` exists in that version, see [pytorch_lightning/utilities/cli.py#L101](https://github.com/Lightning-AI/lightning/blob/1.8.4/src/pytorch_lightning/utilities/cli.py#L101).

**Comment by bparaj** [comment]
Yes, you are right. The image had `1.9.0`. Thank you for the links to the warning messages.
